.github/workflows/main.yaml

```diff
@@ -14,16 +14,12 @@ jobs:
   # Build
   build:
     uses: ./.github/workflows/build.yaml
-  package:
-    needs: build
-    uses: ./.github/workflows/package.yaml
   archive:
     needs: build
     uses: ./.github/workflows/archive.yaml
   upload:
     needs:
       - archive
-      - package
```

Comment:
It looks like the `upload` flow needs `package` to run since it uses the `dist` created.

Reply:
Thanks for catching that! I reverted the changes I made to main.yaml since we now are keeping the package step around.